lib/realtime/view_helpers.rb

```diff
@@ -1,26 +1,17 @@
 module Realtime
   module ViewHelpers
 
-    def realtime_support
-      return render(template: "realtime/realtime_support",
+    def realtime_support(args = {})
+      async = args[:async] ? 'async_' : ''
+      return render(template: "realtime/#{async}realtime_support",
                     layout: nil,
                     locals:
                     { realtime_token: @realtime_token,
                       realtime_domain: @realtime_domain,
                       realtime_server_url: @realtime_server_url,
-                      realtime_user_id: @realtime_user_id }).to_s
-    end
-
-    def realtime_message_handler
```

Comment:
@ayaman I believe removing these view helpers (`realtime_message_handler ` and `realtime_message_console_logger `) and turning them into optional arguments makes the PR backwards-incompatible, requiring a major-version bump (I guess 1.0? With semver it's presently at 0 for the major version, 0.0.12). 

That's not to say your changes aren't a good idea, but anyone who might actually be depending on these still existing during a blindly-trusting `bundle update` run should expect the gem not to break their app.

Maybe you can put these back in, but also before the `return`, you add a `ActiveSupport::Deprecation.warn("deprecation warning")`?